### PR TITLE
Feature: Add i18n module 

### LINKED ISSUE #566: Use chrome.i18n infrastructure to localize PSAT
PSAT, CLI: use the chrome.i18n infrastructure to implement internationalization across PSAT.

### PR DESCRIPTION
**Pull request by mayan-000**
## Description
This PR adds the i18n module which exports the `Singleton` class for handling the request for text translation and parsing the substitutions accordingly.

## Relevant Technical Choices

- [x] Introduce a new i18n package for exporting the class for handling the request for text translation and parsing the substitutions accordingly.
- [x] Create _locales file in i18n package, and have translation of each package, eg. _locales/cli/messages.json
- [x] Add script to merge package's json file into _locales/en/messages.json file
- [x] Write scripts to extract translatable text from files like open-cookie-database.json, PSInfo.json, related_website_sets.json and merge with a single `message.json`.

#### Below pointers will be taken care in upcoming PRs
- [ ] Use translation functions in all package like design-system, common, extension, library-detection, cli-dashboard
- [ ] Get translation done for all required languages.
- [ ] Handle new changes and string updation.
- [ ] Add testcases for functionality.

## Testing Instructions
- This PR only adds i18n infrastructure, it doesn't implement any functions of i18n module.
- The only testing this PR need is that all functionality is working without any breakage.

## Screenshot/Screencast

---

## Checklist

- [x] I have thoroughly tested this code to the best of my abilities.
- [x] I have reviewed the code myself before requesting a review.
- [x] This code is covered by unit tests to verify that it works as intended.
- [x] The QA of this PR is done by a member of the QA team (to be checked by QA).

Fixes #566

### CODE REVIEW

**Comment by mohdsayed** [comment]
@mayank Lets add some test coverage.

File: scripts/update-cookie-db.cjs [hunk 1]
```diff
@@ -81,6 +81,39 @@ const errorHandler = (err) => {
   process.exit(1);
 };
 
+/**
+ * Add keys to the locale file and replace the text with keys in the formattedData.
+ * @param formattedData formatted data
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const addKeysToLocale = async (formattedData) => {
```
Comment (mohdsayed) [inline scripts/update-cookie-db.cjs:89]: Is it being used?

Comment (mayan-000) [reply scripts/update-cookie-db.cjs:89]: The callback is not used currently.

Comment (mohdsayed) [reply scripts/update-cookie-db.cjs:89]: What is purpose of keeping it?

Comment (mayan-000) [reply scripts/update-cookie-db.cjs:89]: It is kept for future usecase when i18n functions will be implemented, so this callback will help creating keys for the db.

Comment (mohdsayed) [reply scripts/update-cookie-db.cjs:89]: Lets add that as a comment.

Comment (mohdsayed) [reply scripts/update-cookie-db.cjs:89]: Merging it for now!

**Review by mohdsayed — APPROVED** [review]

**Pull request merged**